getting-started.md

```diff
@@ -1,49 +1,49 @@
 # Getting Started
 
-Welcome to Kubecost! This page provides commonly used product configurations and feature overviews to help get you up and running after the Kubecost product has been [installed](http://kubecost.com/install). 
-
-__Configuration__  
```

Comment:
Amazing PR! Thank you. The only thing that I noticed is that in markdown new lines are dropped when you remove trailing double spaces in lists like this.

Reply:
Hey @dwbrown2! I don't see it in the markdown app I use (Typora), but I re-added the double spaces that my editor took out. Can you please check if this looks good now?

Reply:
Looks great @minac! Thanks you again so much. I'll go ahead and merge.